src/components/Courses/Courses.tsx

```diff
@@ -31,49 +29,39 @@ const Courses = () => {
 	const dispatch = useDispatch<any>();
 
 	const isCoursesLoaded = useSelector(getIsCoursesLoaded);
-	const isCoursesLoading = useSelector(getIsCoursesLoading);
 	const isAuthorsLoaded = useSelector(getIsAuthorsLoaded);
-	const isAuthorsLoading = useSelector(getIsAuthorsLoading);
 	useEffect(() => {
-		useLoadFullData(
-			dispatch,
-			isCoursesLoaded,
-			isCoursesLoading,
-			isAuthorsLoaded,
-			isAuthorsLoading
-		);
+		useLoadFullData(dispatch, isCoursesLoaded, isAuthorsLoaded);
```

Comment:
It's not allowed to invoke hooks inside useEffect. Wait a second... - it's not a hook :) so please don't name it like a hook to avoid confusion.